__tests__/app.test.js

```diff
@@ -121,31 +118,49 @@ describe("Endpoint: GET /api/articles/:article_id", () => {
   });
 });
 
-// describe("Endpoint: GET /api/articles/:article_id/comments", () => {
-//   test("Returns a status code of 200 when query exists", () => {
-//     return request(app)
-//       .get("/api/articles/1/comments")
-//       .expect(200)
-//   });
-//   test("Returns an array of objects with the correct ids and properties", () => {
-//     return request(app)
-//       .get("/api/articles/1/comments")
-//       .expect(200)
-//       .then((result) => {
-//         console.log(result.body.comments);
-//         result.body.comments.forEach((comment) => {
-//           expect(comment).toMatchObject({
-//             comment_id: expect.any(Number),
-//             votes: expect.any(Number),
-//             created_at: expect.any(String),
-//             author: expect.any(String),
-//             body: expect.any(String),
-//             article_id: expect.any(Number)
-//           });
-//         });
-//       });
-//   });
-// });
+describe("Endpoint: GET /api/articles/:article_id/comments", () => {
+  test("Returns a status code of 200 when query exists", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+  });
+  test("Returns an array of objects with the correct ids and properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.comments.length).toBeGreaterThan(1)
+        result.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+          });
+        });
+      });
+  });
+  test("The comments are sorted in ascending order by their created_at property", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toBeSortedBy("created_at", {
+          ascending: true,
+        });
+      });
+  });
+  test("Returns 400 when passed an invalid id", () => {
+    return request(app)
+      .get("/api/articles/abc/comments")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
+  });
+});
```

Comment:
Add another test - I think you need to have a look at a 200 valid id but there are no comments :)